shell.nix

```diff
@@ -37,7 +37,7 @@ mkDevShell {
     pkgs.jq # for processing data in examples
     pkgs.nixfmt-rfc-style # for formatting nix files
     pkgs.nodejs_18 # for on chain examples
-    pkgs.tinygo # for building wasm
+    pkgs.docker # for building wasm
```

Comment:
I'm no longer seeing tinygo as a dependency, just docker to pull the tinygo container and use it to run tinygo.